### ISSUE TITLE
Add Agency - Northstar Fire Department

### ISSUE THREAD

**Issue by firecaptain55** [GSA/govt-urls#100]
Hello I would like to request the addition of our Fire Department to the list.

Northstarcsd.org
Northstar Fire Department - 910 Northstar Dr. Truckee, CA 96161
Defined as a "special district" located in Truckee, CA. Funded by State and Local tax base. 
Professional full-time paid Fire Department all Paramedics.

Thank you!

**Comment by Cyberamera** [comment]
I agree that the NorthStar Community Services District (NCSD) should be added to the government list. According to CB files, the NCSD appears as a "Special District," and their fire department is one of the many essential community services they provide. Furthermore, since the NCSD is a government-funded organization, it makes sense that it should be included in the government's list of recognized entities.
https://data.ca.gov/dataset/special-districts/resource/afe89e3b-0e84-4f81-965a-4f833d25fbbb?inner_span=True
https://publicpay.ca.gov/Reports/Department.aspx?departmentid=9913&year=2014
https://publicpay.ca.gov/Reports/SpecialDistricts/SpecialDistrict.aspx?entityid=992&year=2020
https://districts.bythenumbers.sco.ca.gov/#!/year/2021/operating/0/entityname/Northstar+Community+Services+District+Financing+Authority/0/districttype2?vis=barChart

**Comment by Cyberamera** [comment]
@jimkreft 
@dawnpm 
@h-m-f-t
